source/_integrations/prometheus.markdown

```diff
@@ -64,6 +64,11 @@ override_metric:
   type: string
   description: Metric name to use instead of unit or default metric. This will store all data points in a single metric.
   required: false
+skip_unknown:
+  type: boolean
+  description: Skip metrics that are in STATE_UNKNOWN to prevent exporting values that could be wrong.
```

Comment:
`STATE_UNKNOWN` is a programming constant and should not end up in end-user documentation.

```suggestion
  description: Skip metrics that are in an unknown state to prevent exporting values that could be wrong.
```